src/main/java/basic/Exercise01.java

```diff
@@ -0,0 +1,20 @@
+package src.main.java.basic;
+
+public class Exercise01 {
+  public int gcd(int x, int y) {
+    if (x == 0) {
+      return y;
+    }
+    if (y == 0) {
+      return x;
+    }
+    if (x == y) {
+      return y;
+    }
+    return gcd(y, x % y);
```

Comment:
Thêm dòng trống vào giữa các khối lệnh


```suggestion
    if (x == 0) {
      return y;
    }
    
    if (y == 0) {
      return x;
    }
    
    if (x == y) {
      return y;
    }
    
    return gcd(y, x % y);
```